tenable/errors.py

```diff
@@ -36,12 +37,13 @@ def __init__(self, resource: str, resource_id: str, filename: str):
         self.resource = str(resource)
         self.resource_id = str(resource_id)
         self.filename = str(filename)
-        self.msg = (f'resource {resource}:{resource_id} '
-                    f'requested file {filename} and has failed.'
-                    )
+        self.msg = (
+            f'resource {resource}:{resource_id} '
+            f'requested file {filename} and has failed.'
+        )
```

Comment:
This change is unrelated to the assets API

Reply:
this had been fixed by pre-commit hooks